app/components/data-table/utils.js

```diff
@@ -14,6 +14,10 @@ const personRegisterDataFormat = [
 ]
  */
 
+const mapOptionsToUrl = (selectedPage, pageSize, sortColumn, sortedAsc) => (
```

Comment:
Looking at this I'm wondering what library options you've got when it comes to constructing URLs, just so you don't have to think about encoding, or how to concat the key value pairs. Might be a tiny bit cleaner.